### PR TITLE
Blockbuilding, overview and github readme

### PR DESCRIPTION
**Pull request by onionhoney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,41 @@
...
+
+- If you have a particular question in mind, it might have been asked by many before. That's why we have created the [FAQ chapter](book.rouxers.com/faqs.html). Try your luck there!
+
+## Organization
+
+At the top level, the reader is organized by the four steps in Roux -- FB, SB, CMLL & LSE, plus a few chapters for generic topics that don't belong to a step. In Blockbuilding, we go over the three fundamental aspects of speedcubing -- **solutions, fingertricks, and lookahead**. For each of these subsections, we first provide an **overview** of what a fast cuber should be ideally doing, and talk about the **progression** -- what do learn at different skill levels. This overview is followed by a curated list of **resources** with commentaries. In CMLL and LSE, this structure is generally followed, with the additional emphasis on sub-step variants and their algorithms. Lastly, while OH and 2H techniques can differ sometimes, they are usually discussed within the same subsections to avoid fragmenting.
```
Comment (V-Wong) [inline README.md:27]: what do learn -> what do I learn

Comment (V-Wong) [inline README.md:27]: to avoid fragmenting -> to avoid fragmentation?

File: src/blockbuilding.md [hunk 2]
```diff
@@ -3,3 +3,31 @@
...
+
+
+<!-- (on a side note: x2y2 means that you're more sensitive to the L/R sides as their colors are fixed.  ) -->
+
+Regarding CN: it gives you 24 options for FB, which is definitely a plus over x2y, but the benefits have not been proven as there has yet to be a world class full CN Rouxer.Some speculate the benefits would be minimal because there is limited time during inspection to analyse all 24 FB options. However, a counterpoint would be that for any first square, you have two choices of last pair to extend to a FB, making the last pair case much better on average. Therefore, once you find a good first square, you can usually settle on it and make better use of your inspection time by looking further into the solve. TL;DR: CN Roux is worthy of exploring, but don't force yourself.
```
Comment (Rouxles) [inline src/blockbuilding.md:29]: Rouxer.Some

File: src/fb.md [hunk 3]
```diff
@@ -1,40 +1,52 @@
...
-If you are interested in the average movecount comparison of FB or FB + DR under different orientations, use the following resource:
+### Relationship to Planning
+Before we go on, note that `planning = solution + tracking`, with solution playing perhaps a more important role. For now stay stuned on getting you solutions right, and we'll cover tracking in the last section.
 
-- [Movecount statistics](https://docs.google.com/spreadsheets/d/1EectP3O_qwQp_2WohbDrlohxhs2ee17sjFfxZr_-D1g/edit#gid=0) - Notice how x2y boosts chance of easy first square compared to y from 60% to 80%.
+### Step II: Improve Your Solutions
 
-## Blockbuilding Strategies
+Next, you want to improve your solutions. There are several aspects you can work on, in no particular order. Not only are easier solutions faster to execute, but the ability to come up with them so will also help bring down your inspection time.
+g.
```
Comment (Rouxles) [inline src/fb.md:28]: what's this line lol

File: src/sb.md [hunk 4]
```diff
@@ -3,11 +3,26 @@
 
 ## Blockbuild Strategy: DR First vs. Freestyle?
 
+DR first is used by most of the world-level rouxers today including Kian Mansour and Sean Villanueva. Most of the existing SB resources are built on top of it. It is easy to make progress with, as it greatly simplifies tracking and pattern recognition for SB blockbuilding.
 
-## Know your Pairs: Pair Progression
+Freestyle, in contrast, allows you to be flexible with how you build the second square. It is a superset of DR first. Strategies include make a pair then insert the remaining edge, or insert any edge then insert the remaining pair.
 
+The consensus is while freestyle potentially leads to better efficiency, it is more demanding of your recognition and tracking skills[^1]. Emperically,   it has yet to be proven that freestyle can serve as a standalone strategy to attain a speed level comparable to those using DR-first.
```
Comment (V-Wong) [inline src/sb.md:10]: Emperically -> Empirically

File: src/overview.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## How to Contribute
+
+If you wish to contribute to the writing, feel free to contact any one of the authors. Please refer to the organization section above when making edits.
+A basic knowledge of `github` and running simple commands from the terminal would be helpful. The workflow is simple enough: Basically, download the binary for [mdbook](https://github.com/rust-lang/mdBook/releases/), clone this repo and make edits to the Markdown files locally, generate the book by running `mdbook serve` on the command line, and submit a pull request when you're done! The book will be automatically generated using a CI workflow once new changes are checked in.
+
+If you have any questions / doubts / suggestions to any content, please also reach out so we can integrate your feedback in future edits. Feedbacks are super important to us!
```
Comment (Rouxles) [inline src/overview.md:22]: Feedback is super important to us

File: README.md [hunk 6]
```diff
@@ -1,4 +1,41 @@
...
+
+## How to Contribute
+
+If you wish to contribute to the writing, feel free to contact any one of the authors. Please refer to the organization section above when making edits.
+A basic knowledge of `github` and running simple commands from the terminal would be helpful. The workflow is simple enough: Basically, download the binary for [mdbook](https://github.com/rust-lang/mdBook/releases/), clone this repo and make edits to the Markdown files locally, generate the book by running `mdbook serve` on the command line, and submit a pull request when you're done! The book will be automatically generated using a CI workflow once new changes are checked in.
+
+If you have any questions / doubts / suggestions to any content, please also reach out so we can integrate your feedback in future edits. Feedbacks are super important to us!
```
Comment (Rouxles) [inline README.md:34]: same thing as the overview

File: src/fb.md [hunk 7]
```diff
@@ -1,40 +1,52 @@
...
 
-There are broadly speaking two families of FB building strategy: square + pair and line + line. See resources below:
+- Influencing
+    - `square + pair` and `line1 + line2` are both two-stage solutions, and the point is we want to find alternative ways to solve the first stage i.e. `square`/`line1` so we get a better second stage case i.e. `pair`/`line2`.
```
Comment (Rouxles) [inline src/fb.md:31]: what is line1/line2 - is that like the E line and the D line?

Comment (Rouxles) [reply src/fb.md:31]: maybe footnotes for these would be nice to have

File: src/fb.md [hunk 8]
```diff
@@ -1,40 +1,52 @@
...
-- [Kavin's Square + Pair Example Solves](https://www.youtube.com/watch?v=YyY_okZ5Fj0)
+- Watch Example Solves
+    - Trainers can only help you so far -- example FB solves remain the best way to learn new ideas. Either watch example solves or go over text reconstructions.
 
-- Louis' Video
+- Develop a Taste for good FB / FS
+    - Taste is about being able to tell an easy FB from a hard one quickly without fully planning it out. This would help you take advantage of all the FB options given by your orientation.
 
+Videos:
 - Kian's Example Solves
```
Comment (Rouxles) [inline src/fb.md:43]: provide a link maybe?

File: src/overview.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+## How to Contribute
+
+If you wish to contribute to the writing, feel free to contact any one of the authors. Please refer to the organization section above when making edits.
+A basic knowledge of `github` and running simple commands from the terminal would be helpful. The workflow is simple enough: Basically, download the binary for [mdbook](https://github.com/rust-lang/mdBook/releases/), clone this repo and make edits to the Markdown files locally, generate the book by running `mdbook serve` on the command line, and submit a pull request when you're done! The book will be automatically generated using a CI workflow once new changes are checked in.
```
Comment (Rouxles) [inline src/overview.md:20]: instead of `github`, maybe [GitHub](https://github.com/rouxers/book) ?

File: src/overview.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+If you have any questions / doubts / suggestions to any content, please also reach out so we can integrate your feedback in future edits. Feedbacks are super important to us!
+
+## Version Log / Roadmap
+
+- version 0.1
+    - This project is still in its early phase as of Sept 22. (v0.1) We hope to get most of the contents up there in a few weeks, so stay tuned for updates!
```
Comment (Rouxles) [inline src/overview.md:27]: i think contents -> content

**Review by Rouxles — COMMENTED** [review]
Just a few small edits to make but other than that it looks fine

File: README.md (hunk 1 above)
Comment (onionhoney) [reply README.md:27]: yes

Comment (onionhoney) [reply README.md:27]: yes

File: src/blockbuilding.md (hunk 2 above)
Comment (onionhoney) [reply src/blockbuilding.md:29]: nice catch

File: src/fb.md (hunk 3 above)
Comment (onionhoney) [reply src/fb.md:28]: lol wat i can't proofread for my life

File: src/overview.md (hunk 9 above)
Comment (onionhoney) [reply src/overview.md:20]: lol github but it links to the book

File: src/overview.md (hunk 10 above)
Comment (onionhoney) [reply src/overview.md:27]: thanks i can't plural

File: src/fb.md (hunk 7 above)
Comment (onionhoney) [reply src/fb.md:30]: good point

**Review by Rouxles — APPROVED** [review]

**Pull request merged**